### ISSUE TITLE
Bugs in 2.3.1.9 (schedule override) and 2.3.1.11

### ISSUE THREAD

**Issue by astevens93** [netmix/radio-station#239]
**Describe the bug**

As you have gathered in past few weeks, I am a heavy user of the plugin - apologies I keep bringing you issues! But hopefully these can be rectified for everyone ahead of an official 2.3.2 release.

**In Dev. 2.3.1.9 and upwards (as this doesn’t seem to get rectified if I upgrade to 2.3.1.11)**

- this is a strange and frustrating one, but I can’t seem to get a show override to work. It saves in wp-admin and everything correctly, but it just does not update on the front end schedule.  It’s a silly one and I know difficult to debug, but have you got this issue elsewhere?  

I then just tried to ‘edit’ that schedule override to see if it worked on another day, and it doesn’t update.  Is there an under lying issue for schedule overrides still?

Last week when I was updating schedule overrides in 2.3.1.9 it worked perfectly. I don't understand why it has broken. Like I say I tried upgrading to 2.3.1.11 and still the same.

**In Dev. 2.3.1.11**

- Month previous on the Schedule Override page is displayed. I.e. you set the over ride date for Saturday 18 July and it shows Saturday 18 June on the WP Admin page for Schedule Override.

- Tabbed schedule view does not load up on the day it is, it defaults to Monday regardless of the day it is. Previous versions loaded tabbed view for that day on initial page load.

**To Reproduce**
1. Add a schedule override - though this seemed to have worked before for me, and now not. I do wonder if I've got a caching problem, I'm not sure.  I did think this got sorted.

**Expected behavior**
Every schedule override would get updated. 

**Screenshots**

Schedule override page in 2.3.1.9 (showing correct July date)

![image](https://user-images.githubusercontent.com/67294458/87287355-ea94bc80-c4f1-11ea-8469-9d09cf743573.png)

Schedule override page in 2.3.1.11 (showing July date displayed as June date)

![image](https://user-images.githubusercontent.com/67294458/87287368-ee284380-c4f1-11ea-9897-cdf128236c9d.png)

**Comment by tonyzeoli** [comment]
Thanks for these reports - in the future, please separate your issues into individual tickets to they can be tracked accordingly.

@majick777 will have a look and let us know if we're seeing this issues too. I'll be testing them today on our demo site to see for myself.

**Comment by majick777** [comment]
@astevens93 I couldn't reproduce any problems with the overrides until I added multiple overrides, then I noticed only one was being honoured in the dev version which led me to find and fix a variable name conflict.
I had already sorted the other two issues over the weekend, I had optimized the month translation but the month index was out by one in that code, so that is sorted now. Also fixed javascript for the day highlighting, as you mentioned in previous ticket.
Test out dev version 2.3.12 all these issues should be resolved now.
